app/assets/javascripts/pageflow/editor/views/theme_view.js

```diff
@@ -0,0 +1,67 @@
+/** @api private */
+pageflow.ThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'pageflow/editor/templates/theme',
+  tagName: 'li',
+
+  ui: {
+    name: '.theme_name',
+    selectButton: '.select_theme',
+    selectedRegion: '.theme_selected'
+  },
+
+  events: {
+    'click .select_theme': function(event) {
+      this.entry.configuration.set('theme_name', this.model.get('name'));
+      pageflow.updateStylesheet('theme');
+    },
+    'mouseenter': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == that.model.get('name');
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName());
+    },
+    'mouseleave': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == pageflow.entry.configuration.attributes.theme_name;
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName({
+        name: pageflow.entry.configuration.attributes.theme_name
+      }));
+    }
```

Comment:
I think the current handling of the active theme (the one that is used in the entry) is fine. The view never has to react to a change, but only has to find out if it is active initially. While I agree that `preview` might be a better name, I would also be totally fine with calling it `selection` if that makes reusing `selectableView` easier.